docs/src/reference/experimental/loop-contracts.md

```diff
@@ -1,13 +1,15 @@
-# Loop Contracts
+# Loop Invariants
 
-Loop contract are used to specify invariants for loops for the sake of extending Kani's *bounded proofs* to *unbounded proofs*.
-A [loop invariant](https://en.wikipedia.org/wiki/Loop_invariant) is an expression that holds upon entering a loop and after every execution of the loop body.
+## Introduction
+
+Loop-invariants are used to specify invariants for loops for the sake of extending Kani's *bounded proofs* to *unbounded proofs*.
+A [loop-invariant](https://en.wikipedia.org/wiki/Loop_invariant) is an expression that holds upon entering a loop and after every execution of the loop body.
 It captures something that does not change about every step of the loop.
 
 It is worth revisiting the discussion about [bounded proof](../../tutorial-loop-unwinding.md#bounded-proof) and
 [loop unwinding](../../tutorial-loop-unwinding.md#loops-unwinding-and-bounds). In short, bounds on the number of times Kani unwinds loops also bound the size of inputs,
 and hence result in a bounded proof.
-Loop contracts are used to abstract out loops as non-loop blocks to avoid loop unwinding, and hence remove the bounds on the inputs.
+Loop-invariants are used to abstract out loops as non-loop blocks to avoid loop unwinding, and hence remove the bounds on the inputs.
```

Comment:
Please replace all occurrences of "Loop-invariants" with "Loop invariants" (i.e. remove the dash).